fs/io_zenfs.cc

```diff
@@ -705,6 +713,36 @@ void ZoneFile::ReplaceExtentList(std::vector<ZoneExtent*> new_list) {
   extents_ = new_list;
 }
 
+void ZoneFile::SetLinkName(const std::string& linkf) {
```

Comment:
AddLinkName sounds good to me

Reply:
> Can we rename this one to AddLinkName?
Ok, AddLinkName better encapsulates what is happening..
 
> I would suggest to remove ZoneFile::Rename() as part of this PR.
We can remove it, but I think it is a good way to indicate what is happening instead of just doing 
`linkfiles_[0] = newname`. So it does not hurt to keep it. Lets keep it.

Reply:
`Rename()` is only called in two locations, both of which are redundant. It can just be removed.